pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -147,6 +147,9 @@ func (p *criStatsProvider) listPodStats(updateCPUNanoCoreUsage bool) ([]statsapi
 			klog.V(5).ErrorS(err,
 				"CRI implementation must be updated to support ListPodSandboxStats if PodAndContainerStatsFromCRI feature gate is enabled. Falling back to populating with cAdvisor; this call will fail in the future.",
 			)
+		} else {
```

Comment:
nit: we can invert these conditions:
```
if err == nil {
    return result, nil
}
s, ok ...
```

Reply:
Of course, I originally wanted to write it this way, but I felt that the code changes would be a bit more. I will update it.